lib/views/pages/login_signup/register_form.dart

```diff
@@ -286,9 +286,10 @@ class RegisterFormState extends State<RegisterForm> {
                           prefixIcon: Icon(Icons.lock, color: Colors.white),
                           suffixIcon: FlatButton(
                             onPressed: _toggle,
-                            child: Icon(_obscureText
-                                ? Icons.visibility_off
-                                : Icons.visibility,
+                            child: Icon(
```

Comment:
Please, do not add any changes other than part of the issue.
Revert all these changes to merge this PR.